packages/did/README.md

```diff
@@ -1,6 +1,6 @@
 # @tangleid/did
 
-Utilities for Decentralized Identifiers(DIDs) registration and resolve.
+Utilities for decentralized identifiers encoding and decoding.
```

Comment:
Should be "Utilities for the coding of decentralized identifiers"

Reply:
Fixed.